common/src/chain/config.rs

```diff
@@ -442,34 +431,7 @@ pub fn create_unit_test_config() -> ChainConfig {
     let genesis_block = create_unit_test_genesis(Destination::AnyoneCanSpend);
     let genesis_block_id = genesis_block.get_id();
 
-    let target_block_spacing = DEFAULT_TARGET_BLOCK_SPACING;
-    let coin_decimals = MAINNET_COIN_DECIMALS;
-
-    let expected_total_block_subsidy =
-        (Amount::from_fixedpoint_str(MAINNET_TOTAL_SUPPLY, coin_decimals).unwrap()
-            - Amount::from_fixedpoint_str(MAINNET_COIN_PREMINE, coin_decimals).unwrap())
-        .unwrap();
-
-    let subsidy_per_block_in_year_n_str = [
-        (0, "202"),
-        (1, "151"),
-        (2, "113"),
-        (3, "85"),
-        (4, "64"),
-        (5, "48"),
-        (6, "36"),
-        (7, "27"),
-        (8, "20"),
-        (9, "15"),
-        (10, "0"),
-    ];
-
-    let emission_schedule = emission_schedule_from_yearly(
-        &subsidy_per_block_in_year_n_str,
-        &target_block_spacing,
-        coin_decimals,
-        Some(expected_total_block_subsidy),
-    );
+    let (target_block_spacing, coin_decimals, emission_schedule) = mainnet_emission_schedule();
```

Comment:
This needs to be overhauled in a different way. There has to be a module for the emission schedule and all its details (I mentioned that in a comment somewhere in this file). I don't mind making this change now, but this is just temporary.